packages/libs/api/icy-graphql-client/testSetup/pollyTestSetup.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint @typescript-eslint/ban-ts-comment: "off" */
```

Comment:
this can be moved to a separate library like i did [over here](https://github.com/quiknode-labs/quicknode-typescript/blob/main/libs/tsutils/src/test/pollyTestSetup.ts).  Not a big deal if you want to punt this for now though.